pkg/cloudprovider/gcp/object_store.go

```diff
@@ -94,16 +113,18 @@ func (o *ObjectStore) Init(config map[string]string) error {
 
 	o.googleAccessID = jwtConfig.Email
 	o.privateKey = jwtConfig.PrivateKey
+	return nil
+}
 
-	client, err := storage.NewClient(context.Background(), option.WithScopes(storage.ScopeReadWrite))
-	if err != nil {
-		return errors.WithStack(err)
+func (o *ObjectStore) initFromComputeEngine(config map[string]string) error {
+	var err error
+	var ok bool
+	o.googleAccessID, ok = config["serviceAccount"]
+	if !ok {
+		return errors.Errorf("could not find the service account name from the compute engine credentials")
```

Comment:
let's update this error to clarify that `serviceAccount` is expected to be provided as a config item in the `BackupStorageLocation`'s config.